pandas/core/groupby.py

```diff
@@ -1055,6 +1056,60 @@ def tail(self, n=5):
         tail = obj[in_tail]
         return tail
 
+    def pipe(self, func, *args, **kwargs):
+        """
+        Apply func(self, \*args, \*\*kwargs)
+
+        .. versionadded:: 0.16.3
+
+        Parameters
+        ----------
+        func : function
+        function to apply to the %(klass)s.
```

Comment:
Can you remove the ``%(klass)s`` etc (that is leftover from the dataframe/series method)